_layouts/about.html

```diff
@@ -4,7 +4,7 @@
 <div id="main-directors-list">
     <h2>Meet our Company!</h2>
     {% for staff in page.main-staff %}
-        <img src={{staff.image-path}} alt={{staff.name}}></img>
+        <img src={{staff.image-path}} alt={{staff.name}}>
```

Comment:
These still need to be wrapped in strings. Otherwise you see things like `<img alt="tyler" small>` where it thinks small is now a property of what we're fixing.